File: C# PowerPoint Examples/PropertiesApiDemo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Spire.Cloud.Powerpoint.Sdk.Client;
using Spire.Cloud.Powerpoint.Sdk.Api;
using Spire.Cloud.Powerpoint.Sdk.Model;

namespace CloudPowerpoint
{
    class PropertiesApiDemo
    {
        static String appId = "your id";
        static String appKey = "your key";
        static String baseUrl = "https://api.e-iceblue.cn";
        static Configuration configuration = new Configuration(baseUrl, appId, appKey);
        static PropertiesApi propertiesApi = new PropertiesApi(configuration);
        public static void DeletePptDocumentProperties()
        {
            string name = "deletePptDocumentProperties.pptx";
            string password = null;
            string folder = "input";
            string storage = null;
            var response = propertiesApi.DeletePptDocumentProperties(name, password, folder, storage);
        }
        public static void DeletePptDocumentProperty()
        {
            string name = "deletePptDocumentProperty.pptx";
            string propertyName = "Author";
            string password = null;
            string folder = "input";
            string storage = null;
            var response = propertiesApi.DeletePptDocumentProperty(name, propertyName, password, folder, storage);
        }
        public static void GetPptDocumentProperties()
        {
            string name = "getPptDocumentProperties.pptx";
            string password = null;
            string folder = "input";
            string storage = null;
            var response = propertiesApi.GetPptDocumentProperties(name, password, folder, storage);
        }
        public static void GetPptDocumentProperty()
        {
            string name = "getPptDocumentProperty.pptx";
            string propertyName = "Author";//Available values : Company, Format, LastSavedTime, Title
            string password = null;
            string folder = "input";
            string storage = null;
            var response = propertiesApi.GetPptDocumentProperty(name, propertyName, password, folder, storage);
        }
        public static void SetPptDocumentProperties()
        {
            string name = "setPptDocumentProperties.pptx";
            string password = null;
            string folder = "input";
            string storage = null;

            List<DocumentProperty> propertyList = new List<DocumentProperty>();
            DocumentProperty property1 = new DocumentProperty();
            property1.Name="Keywords";
            property1.Value="Set document properties.";
            property1.BuiltIn=false;
            propertyList.Add(property1);
            DocumentProperty property2 = new DocumentProperty();
            property2.Name = "Author";
            property2.Value = "eiceblue";
            property2.BuiltIn = false;
            propertyList.Add(property2);
            DocumentProperty property3 = new DocumentProperty();
            property3.Name = "Company";
            property3.Value = "冰蓝";
            property3.BuiltIn = false;
            propertyList.Add(property3);
            DocumentProperty property4 = new DocumentProperty();
            property4.Name = "LastSavedBy";
            property4.Value = "123456@iCloud.com";
            property4.BuiltIn = false;
            propertyList.Add(property4);
            DocumentProperty property5 = new DocumentProperty();
            property5.Name = "SharedDoc";
            property5.Value = "true";
            property5.BuiltIn = false;
            propertyList.Add(property5);
            DocumentProperties properties = new DocumentProperties(propertyList);

            var response = propertiesApi.SetPptDocumentProperties(name, properties, password, folder, storage);
        }
        public static void SetPptDocumentProperty()
        {
            string name = "setPptDocumentProperty.pptx";
            string propertyName = "Keywords";
            DocumentProperty property = new DocumentProperty("Keywords", "E-iceblue", false);
            string password = null;
            string folder = "input";
            string storage = null;
            var response = propertiesApi.SetPptDocumentProperty(name, propertyName, property, password, folder, storage);
        }
    }
}

File: Java PowerPoint Examples/PropertiesApiDemo.java
import spire.cloud.powerpoint.sdk.*;
import spire.cloud.powerpoint.sdk.api.PropertiesApi;
import spire.cloud.powerpoint.sdk.model.*;

public class PropertiesApiDemo {
    static String appId = "your id";
    static String appKey = "your key";
    static String baseUrl = "https://api.e-iceblue.cn";
    static Configuration configuration = new Configuration(appId, appKey,baseUrl);
    static PropertiesApi propertiesApi = new PropertiesApi(configuration);

    public static void deletePptDocumentProperties() throws ApiException {
        String name = "deletePptDocumentProperties.pptx";
        String password = null;
        String folder = "input";
        String storage = null;
        DocumentProperties response = propertiesApi.deletePptDocumentProperties(name, password, folder, storage);
    }
    public static void deletePptDocumentProperty() throws ApiException {
        String name = "deletePptDocumentProperty.pptx";
        String propertyName = "Author";
        String password = null;
        String folder = "input";
        String storage = null;
        DocumentProperties response = propertiesApi.deletePptDocumentProperty(name, propertyName, password, folder, storage);
    }
    public static void getPptDocumentProperties() throws ApiException {
        String name = "getPptDocumentProperties.pptx";
        String password = null;
        String folder = "input";
        String storage = null;
        DocumentProperties response = propertiesApi.getPptDocumentProperties(name, password, folder, storage);
    }
    public static void getPptDocumentProperty() throws ApiException {
        String name = "getPptDocumentProperty.pptx";
        String propertyName = "Author";//Available values : Company, Format, LastSavedTime, Title
        String password = null;
        String folder = "input";
        String storage = null;
        DocumentProperty response = propertiesApi.getPptDocumentProperty(name, propertyName, password, folder, storage);
    }
    public static void setPptDocumentProperties() throws ApiException {
        String name = "setPptDocumentProperties.pptx";
        String password = null;
        String folder = "input";
        String storage = null;

        java.util.ArrayList<DocumentProperty> propertyList = new java.util.ArrayList<>();
        DocumentProperty property1 = new DocumentProperty();
        property1.setName("Keywords");
        property1.setValue("Set document properties.");
        property1.setBuiltIn(false);
        propertyList.add(property1);
        DocumentProperty property2 = new DocumentProperty();
        property2.setName( "Author");
        property2.setValue("eiceblue");
        property2.setBuiltIn(false);
        propertyList.add(property2);
        DocumentProperty property3 = new DocumentProperty();
        property3.setName("Company");
        property3.setValue("iceblue");
        property3.setBuiltIn(false);
        propertyList.add(property3);
        DocumentProperty property4 = new DocumentProperty();
        property4.setName("LastSavedBy");
        property4.setValue("123456@iCloud.com");
        property4.setBuiltIn(false);
        propertyList.add(property4);
        DocumentProperty property5 = new DocumentProperty();
        property5.setName("SharedDoc");
        property5.setValue("true");
        property5.setBuiltIn(false);
        propertyList.add(property5);
        DocumentProperties properties = new DocumentProperties();
        properties.list(propertyList);
        DocumentProperties response = propertiesApi.setPptDocumentProperties(name, properties, password, folder, storage);
    }
    public static void setPptDocumentProperty() throws ApiException {
        String name = "setPptDocumentProperty.pptx";
        String propertyName = "Keywords";
        DocumentProperty property = new DocumentProperty();
        property.setName("Keywords");
        property.setValue("SetPptDocumentProperty_pptx");
        property.setBuiltIn(false);
        String password = null;
        String folder = "input";
        String storage = null;
        DocumentProperty response = propertiesApi.setPptDocumentProperty(name, propertyName, property, password, folder, storage);
    }
}
